docs/admin/predeployment_check.md

```diff
@@ -239,10 +239,32 @@ running as a systemd service. Default ulimit settings vary between OSes.
 
 For RPM installations, the service will typically be launched by systemd and
 the limit is pre-set to unlimited in the `daos_server.service` unit file:
-https://github.com/daos-stack/daos/blob/master/utils/systemd/daos_server.service#L16.
+https://github.com/daos-stack/daos/blob/master/utils/systemd/daos_server.service
+
 Note that values set in `/etc/security/limits.conf` are ignored by services
 launched by systemd.
 
 For non-RPM installations where `daos_server` is launched directly from the
 commandline, limits should be adjusted in `/etc/security/limits.conf` as per
-https://software.intel.com/content/www/us/en/develop/blogs/best-known-methods-for-setting-locked-memory-size.html.
+https://access.redhat.com/solutions/61334 (RHEL specific document but
+the instructions apply to most Linux distributions).
+
+## Socket receive buffer size
+
+Low socket receive buffer size can cause SPDK to fail and emit the following
+error (receive buffer size is required to be above 1MB):
+
+```bash
+daos_engine:1 pci_event.c:  68:spdk_pci_event_listen: *ERROR*: Failed to set socket
+ option
+```
+
+The socket receive buffer size does not need to be manually adjusted if
+`daos_server` has been installed using an RPM package (as the settings
+will be applied automatically on install).
+
+For non-RPM installations where `daos_server` has been built from source,
+`rmem_default` and `rmem_max` settings should be set to >= 1MB as per
+https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/5/html/tuning_and_optimizing_red_hat_enterprise_linux_for_oracle_9i_and_10g_databases/sect-oracle_9i_and_10g_tuning_guide-adjusting_network_settings-changing_network_kernel_settings
```

Comment:
It might be nicer to put this long, ugly URL into a \[...\]() with some nice text instead.